tests/vip-helpers/test-vip-elasticsearch.php

```diff
@@ -12,6 +12,9 @@ public function test__vip_elasticsearch_filter_ep_index_name() {
 
 		define( 'USE_VIP_ELASTICSEARCH', true );
 
+		// Hack to get around the constant not being defined early enough...there is probably a proper PHPUnit way to do that
```

Comment:
There's no clean way to do this since the constant is defined in the bootstrap. We'd have to remove it from there to be able to get more control around these tests.